packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -488,22 +488,35 @@ export class Repository extends Construct implements IRepository {
 
     this.version = props.version;
 
-    // Set up the Filesystem of the repository
-    if (props.fileSystem !== undefined) {
-      this.fileSystem = props.fileSystem;
-    } else {
-      this.efs = new EfsFileSystem(this, 'FileSystem', {
+    this.fileSystem = props.fileSystem ?? (() => {
+      const fs = new EfsFileSystem(this, 'FileSystem', {
         vpc: props.vpc,
         vpcSubnets: props.vpcSubnets ?? { subnetType: SubnetType.PRIVATE },
         encrypted: true,
         lifecyclePolicy: EfsLifecyclePolicy.AFTER_14_DAYS,
         removalPolicy: props.removalPolicy?.filesystem ?? RemovalPolicy.RETAIN,
         securityGroup: props.securityGroupsOptions?.fileSystem,
       });
-      this.fileSystem = new MountableEfs(this, {
-        filesystem: this.efs,
+
+      const accessPoint = fs.addAccessPoint('AccessPoint', {
+        path: '/DeadlineRepository',
+        createAcl: {
+          ownerUid: '10000',
+          ownerGid: '10000',
+          permissions: '770',
+        },
+        posixUser: {
+          uid: '10000',
+          gid: '10000',
+        },
       });
-    }
+
+      return new MountableEfs(this, {
+        filesystem: fs,
+        accessPoint,
+        extraMountOptions: [`accesspoint=${accessPoint.accessPointId}`],
```

Comment:
Looks like we can remove this since we're adding it in `MountableEfs` already: https://github.com/aws/aws-rfdk/pull/339/files#diff-4add095eaf52008f1b5fb6476d138899d4583b2dc36b57bddd9e223a1ea22fd8R123-R128

Reply:
Good catch. removed